frontend/src/enums/ErrorMessages.ts

```diff
@@ -0,0 +1,3 @@
+export const ERROR_MESSAGES = {
```

Comment:
Search up how to write  an enum. THis is technically fine but it is still not an Enum